admin-dev/themes/new-theme/js/pages/category/index.js

```diff
@@ -89,7 +89,7 @@ $(() => {
   new Serp(
     {
       container: '#serp-app',
-      defaultTitle: 'input[name^="category[name]',
+      defaultTitle: 'input[name^="category[name]"]',
       watchedTitle: 'input[name^="category[meta_title]',
```

Comment:
@PierreRambaud Is it normal that another css selectors has not the ending quote & square bracket ?

Reply:
Oh no, you're right, I'll update this